src/components/SvgMousePosition/index.test.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import * as throttleDebounce from 'throttle-debounce';
+import SvgMousePosition from './component';
+
+const svg = document.createElement('svg');
```

Comment:
The comments I made at `NodeMovingPlaceholder/index.test.js` apply here as well.